### DISCUSSION TITLE
"Progress" is not listed anymore!

### DISCUSSION THREAD

**Discussion by Luram123** [MuiseDestiny/zotero-style#850] [General]
I'm using 7.0.0-beta.56+9edfcba9a (64-bit).
Today Ethereal Style stopped showing the progress column and when I checked the settings, the function was gone.

**Comment by MuiseDestiny** [comment]
use https://gitee.com/MuiseDestiny/plugins/blob/master/zotero-style.xpi

**Reply by unaexiliada** [reply]
I tried this and it also didn't work.
I added the plugin and progress was working perfectly, until I tried to convert the "bar" into a circle and everything dissapeared. When I dissabled "progress" from the preferences of the plugin, everything appeared again, but of course I cannot use the progress tab.

**Reply by MuiseDestiny** [reply]
only support `bar` `line` `stack`

**Reply by unaexiliada** [reply]
thank you, I realized that after I made the mistake! But there's no way to undo it. I tried removing and installing again, using an older version and nothing seems to work. If I enable "progress", everything dissapears, I cannot see any of my files, nor the tabs. 
Do you know what can be done?
<img width="1511" alt="Captura de pantalla 2024-02-02 a la(s) 23 34 51" src="https://github.com/MuiseDestiny/zotero-style/assets/158686291/fea71216-a5e4-4a9d-b141-f2b0099c4bfe">

**Reply by MuiseDestiny** [reply]
![image](https://github.com/MuiseDestiny/zotero-style/assets/51939531/486fd27e-fb90-4e68-8c87-ad68730885ad)

**Reply by unaexiliada** [reply]
thank you! it works perfectly!
